code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -1,14 +1,18 @@
 /obj/item/grenade/flashbang
 	name = "flashbang"
+	desc = "A less-than-lethal grenade designed for crowd control. Upon detonation it produces an extremely intense flash of light that blinds targets, \
+	and releases a shockwave that induces temporary deafness and causes a loss of balance by disturbing the fluid canals of the inner ear. Both effects can be negated with proper protective equipment."
```

Comment:
This is incredibly convoluted and should at the very least be moved to an extended examine

Reply:
Shortened.